Assets/Scripts/Abilities/PlayerBasedAreaAbility.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "Ability/Player Based Area Ability")]
+public class PlayerBasedAreaAbility : AbilityBase
+{
+
+    public float diameter = 2f;
+    
+
+    public override void Initialize()
+    {
+
+    }
+
+    public override void Activate()
+    {
+
+    }
+}
```

Comment:
we don;t need this as a player based area ability is just an area ability with the center being the player. if we need the distinction we could just make this inherit from targetAreaAbility with the specified caveat that its centered on the [;ayer